python/enthic/app.py

```diff
@@ -178,23 +172,73 @@ def pre_cast_integer(probe):
     return str(probe) if probe.__class__ is int else probe if probe.isnumeric() is True else None
 
 
-def result_array(probe, limit, offset=0):
+def ape_list(ape_code):
+    """
+    APE code are not save in the original format.
+    This function returns the corresponding code in MySQL or None
+    if the code does not exist
+
+        :param ape_code : the list of APE for he filter
+        :param APE_CODE : all known APE code
+
+        :return: list of corresponding  APE_CODE in MySQL
+    """
+    ape = list()
+    for ap in ape_code.split(","):
+        for i in APE_CODE:
+            if re.match(ap,APE_CODE[i][0]):
+                ape.append(i)
+    if len(ape) == 0:
```

Comment:
Don't call len function it is useless:
```
>>> if []:
	print('true')

	
>>> if [1]:
	print('true')

	
true
```

Reply:
Okay

Reply:
I'll apply your recommendations and push it tomorrow